scripts/generate-pkg-config.js

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env node
+
+/*
+ * Morgan Stanley makes this available to you under the Apache License,
+ * Version 2.0 (the "License"). You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership. Unless required by applicable law or agreed
+ * to in writing, software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+// Platform mappings for prebuilds
+const PLATFORM_MAPPINGS = {
```

Comment:
If you export this you can use it in the tests instead of the array of platforms.

Reply:
I created an issue - #22 22